hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/TestHoodieDeltaStreamerSchemaEvolutionExtensive.java

```diff
@@ -200,8 +237,17 @@ protected static Stream<Arguments> testArgs() {
         for (Boolean multiLogFiles : new Boolean[]{false, true}) {
           for (Boolean shouldCluster : new Boolean[]{false, true}) {
             for (String tableType : new String[]{"COPY_ON_WRITE", "MERGE_ON_READ"}) {
-              if (!multiLogFiles || tableType.equals("MERGE_ON_READ")) {
-                b.add(Arguments.of(tableType, shouldCluster, false, rowWriterEnable, addFilegroups, multiLogFiles));
+              for (Boolean isSparkRecord : new Boolean[]{false, true}) {
+                if (!multiLogFiles || tableType.equals("MERGE_ON_READ")) {
+                  b.add(Arguments.of(
+                      tableType,
+                      shouldCluster,
+                      false,
+                      rowWriterEnable,
+                      addFilegroups,
+                      multiLogFiles,
+                      isSparkRecord));
```

Comment:
Put on the same line.

Reply:
Done. What is the limit of the line length?

Reply:
Anything below 120 is ok. Hard limit is 200.

Reply:
Ok. 200 is bad for my neck. I will my best for 120. ;)